.github/workflows/release.yml

```
@@ -13,10 +13,10 @@ jobs:
             - name: Checkout Repo
               uses: actions/checkout@master
 
-            - name: Setup Node.js 10.x
+            - name: Setup Node.js 18
               uses: actions/setup-node@master
               with:
-                  node-version: 10.x
+                  node-version: '18'
 
```

I know this isn't a change in your PR, but I'd prefer to see these pinned to a major version.

```suggestion
            - name: Checkout Repo
              uses: actions/checkout@v4

            - name: Setup Node.js 18
              uses: actions/setup-node@v4
              with:
                  node-version: 18
```

as per https://github.com/actions/setup-node?tab=readme-ov-file#usage & https://github.com/actions/checkout?tab=readme-ov-file#usage